src/main/java/com/domychoresserver/domychoresserver/Payload/Response/JwtResponse.java

```diff
@@ -20,6 +20,11 @@
     private String email;
     private List<String> roles;
 
-    public JwtResponse(String jwt, String id, String username, String email, List<String> roles) {
+    public JwtResponse(String token, String id, String username, String email, List<String> roles) {
```

Comment:
The user's fields are:
- id
- email
- firstName
- lastName
- avatar (need discuss, how to store it)
- role